addons/mail/views/mail_mail_views.xml

```diff
@@ -39,7 +39,7 @@
                         </group>
                         <notebook>
                             <page string="Body" name="body">
-                            <field name="body_html" widget="html"
+                            <field name="body_html"
```

Comment:
As the name clearly states, it is actually a Text field :D (and should stay as it is because lot of reasons).